cmd/horcrux/cmd/root.go

```diff
@@ -30,13 +30,14 @@ func Execute() {
 
 func init() {
 	cobra.OnInitialize(initConfig)
-	rootCmd.PersistentFlags().StringVar(&cfgFile, "config", "", "config file (default is $HOME/.horcrux/config.yaml)")
+	rootCmd.PersistentFlags().StringVar(&cfgFile, "home", "", "directory for config and data (default is $HOME/.horcrux/)")
```

Comment:
One note here: we are using the `cfgFile` var to store the home. This could be confusing. Maybe change the `cfgFile` var to `homeDir` or something?

Reply:
Yeah that makes sense, I forgot to leave a note and I didn't change the name due to having to tweak some statements in `config.go`

I went ahead and refactored it and changed some parts in `test/signer_test.go` and `config.go`so that `--home` functions as intended